server-spi-private/src/main/java/org/keycloak/broker/provider/mappersync/ConfigSyncEventListener.java

```diff
@@ -51,12 +51,7 @@ public void onEvent(ProviderEvent event) {
                 LOG.debugf("Synchronizer %s matches event: %s", configSynchronizer, event);
 
                 if (realmMappers == null) {
-                    /*
-                     * an event always refers to just one realm, so we can use an arbitrary synchronizer to extract the
-                     * realm
-                     */
-                    RealmModel realm = configSynchronizer.extractRealm(event);
-                    realmMappers = realm.getIdentityProviderMappersStream()
+                    realmMappers = configSynchronizer.getKeycloakSession(event).identityProviders().getMappersStream()
```

Comment:
Do we have a follow-up issue to sort out this usage of `getMappersStream`?

Reply:
This one? https://github.com/keycloak/keycloak/issues/32233